spark/homescreens.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I feel like it makes sense to have `/distributions/spark/` or `/customizations/spark/`, we can have

distributions
 + spark
 + example
 + more to come